aten/src/ATen/native/native_functions.yaml

```diff
@@ -1942,6 +1946,9 @@
 - func: max_values.names(Tensor self, Dimname[1] dim, bool keepdim=False) -> Tensor
   variants: function, method
 
+- func: amax.names(Tensor self, Dimname[1] dim, bool keepdim=False) -> Tensor
```

Comment:
If you're not supporting named tensors in this first PR (which is totally reasonable, btw), better not add a named tensor signature.

Reply:
Yeah, I agree. And I will remove these signatures. `max_values` pretend to be supporting named tensor, but the implementation is only
```C++
TORCH_CHECK(false, "NYI: max_values with names");
return at::max_values(self, dimnames_to_positions(self, dims), keepdim);
```

Reply:
Pretending is lame.